src/Utilities.mss

```diff
@@ -1,41 +1,59 @@
 function MSplitString (string, delimiter) {
     //$module(Utilities.mss)
     /*
-        The default Splitstring method is buggy,
-        so I've re-implemented it here.
+        The default SplitString method returns a TreeNode Array which is hard
+        to handle and can lead to Sibelius crashes.  This is a more friendly
```

Comment:
Double space.

Reply:
A habit of mine to add double spaces after full stops. I can change it.